lib/IAMClient.js

```diff
@@ -290,17 +292,25 @@ class VaultClient {
      * @returns {undefined}
      */
     updateAccountQuota(accountName, quota, callback) {
-        const conv = Number.isNaN(Number.parseInt(quota, 10));
-        assert(typeof accountName === 'string' && accountName !== '',
-            'accountName is required');
-        assert(typeof quota === 'number' && !conv
-            && quota >= 0, 'Quota must be a non-negative number');
-        this.request('POST', '/', true, callback, {
+        const disableValidation = this.disableValidation;
```

Comment:
why alias the variable?

Reply:
fixed here : https://github.com/scality/vaultclient/pull/442/commits/29e09a955ef3aef0716b15f9f096a2739e6f0c04